web-console/src/utils/index.tsx

```diff
@@ -20,3 +20,4 @@ export * from './general';
 export * from './druid-query';
 export * from './query-manager';
 export * from './rune-decoder';
+export * from './table';
```

Comment:
please give this a slightly less generic name. in fact it should really reflect the name of the class inside of it so `table-column-selection-handler`